packages/backend/src/config/app.ts

```diff
@@ -138,6 +138,8 @@ export const Config = {
   signatureSecret: process.env.SIGNATURE_SECRET, // optional
   signatureVersion: envInt('SIGNATURE_VERSION', 1),
 
+  apiSecret: process.env.API_SECRET, // optional
```

Comment:
My understanding is that the `apiSecret` is an optional environment variable that secures the admin api if provided, and if not, it's unsecured. I'm just wondering if we want it to be secure by default instead? I guess this way is more backwards compatible I just find my initial assumptions were secure by default.

Noticed when I was surprised the integration tests didn't fail.

Reply:
I couldn't remember where I heard this, but I had thought I'd heard at some point that securing the admin apio should be optional.

Reply:
Yeah I recall that as well. Just wondering if it should be opt-in or opt-out. It's not necessarily a suggestion, just thinking out loud. 

I generally assume people should have to explicitly opt-out of the more secure way but perhaps in this specific case it doesnt matter? Since its currently insecure and not meant to be exposed to the internet. I think the configuration (how you have it now) is probably simpler than enabling by default (by expecting `apiSecret` unless some additional env var is set to bypass it or something).